azure-mgmt-network/src/main/java/com/azure/management/network/implementation/RouteTableImpl.java

```diff
@@ -132,11 +132,6 @@ protected void beforeCreating() {
         this.inner().withRoutes(innersFromWrappers(this.routes.values()));
     }
 
-    @Override
-    protected void afterCreating() {
-        initializeChildrenFromInner();
```

Comment:
Is it called somewhere instead?

Reply:
The `initializeChildrenFromInner` is call before `afterCreating` in `GroupableParentResourceImpl.createResourceAsync`, https://github.com/azure/azure-libraries-for-java/blob/95bc556/azure-mgmt-resources/src/main/java/com/azure/management/resources/fluentcore/arm/models/implementation/GroupableParentResourceImpl.java#L57